test/util/a11y/validate.js

```diff
@@ -68,19 +80,19 @@ export const mountReact = async function (node, id) {
  *                 {Object} `rules` - set properties for rules
  * @param {Promise} wrapper of the react component
  */
-export const testReact = async function (node, options = {}) {
-    const wrapper = mountReact(node, divId);
-    await test(`#${divId}`, options);
+// TODO: resolve issue where failing tests do not pass wrapper and so cannot be cleaned up correctly
+export const testReact = async function(node, options = {}) {
+    const wrapper = mountReact(node, A11Y_ROOT_ID);
+    await test(`#${A11Y_ROOT_ID}`, options);
     return wrapper;
 };
 
-
 /**
- * Helper function to use with `testReact` to unmount the div wrapper
+ * Helper function to use with `testReact` to unmount a dom node, defaults to the root a11y node
  */
-export const afterEach = function () {
-    const div = document.querySelector(`#${divId}`);
-    if (div) {
-        div.remove();
+export const unmount = function(id = A11Y_ROOT_ID) {
```

Comment:
rename to `unmount` to be clear about what it does. 
set param defaults directly